tests/OmniSharp.Tests/DotNetCliServiceFacts.cs

```diff
@@ -7,9 +8,20 @@ namespace OmniSharp.Tests
 {
     public class DotNetCliServiceFacts : AbstractTestFixture
     {
+        private const string DotNetVersion = "3.1.401";
+        private int Major { get; }
+        private int Minor { get; }
+        private int Patch { get; }
+        private string Release { get; }
+
         public DotNetCliServiceFacts(ITestOutputHelper output)
             : base(output)
         {
+            var version = SemanticVersion.Parse(DotNetVersion);
+            Major = version.Major;
+            Minor = version.Minor;
+            Patch = version.Patch;
+            Release = version.Release;
```

Comment:
This should make future SDK updates easier since the version string can be updated with a replace all.